src/dashboard/src/components/api/views.py

```diff
@@ -508,6 +485,8 @@ def copy_metadata_files_api(request):
     paths = request.POST.getlist('source_paths[]')
     return filesystem_ajax_views.copy_metadata_files(sip_uuid, paths)
 
+# TODO should this have auth?
+@_api_endpoint(expected_methods=['GET'])
 def get_levels_of_description(request):
```

Comment:
This had no auth or verb restriction previously - needs to be tested to see if it still works with external consumers.

Also, this returns a list, not a json (with eg a 'results' list) - should it be changed for consistency?